src/main/java/io/vertx/gsoc2023/qotd/QuoteOfTheDayVerticle.java

```diff
@@ -66,21 +66,16 @@ private void setupPost(Router router) {
         JsonObject body = ctx.body().asJsonObject();
         if (body != null && body.containsKey("text")) {
           pgPool.preparedQuery("INSERT INTO quotes (text, author) VALUES ($1, $2) RETURNING *")
-            .execute(Tuple.of(body.getString("text"), body.getString("author", "Unknown")),
-              ar -> {
-                if (ar.succeeded()) {
-                  var result = ar.result();
-                  Row row = result.iterator().next();
-                  JsonObject asJson = row.toJson();
-                  // Send inserted quote to websockets listening to /realtime
-                  for (String socketWriterHandlerID : socketsWriterHandlerIDs) {
-                    vertx.eventBus().send(socketWriterHandlerID, asJson.toBuffer());
-                  }
-                  ctx.json(asJson);
-                } else {
-                  ctx.fail(ar.cause());
-                }
-              });
+            .execute(Tuple.of(body.getString("text"), body.getString("author", "Unknown")))
+            .onSuccess(result -> {
```

Comment:
what about the error case then?

Reply:
I'm guessing it will return a 500 status code which is what I was manually doing anyways. No more guessing though, added a test (which dirtied up a bit the test class, if you have a better alternative tell me)

Reply:
Weird, passes locally but it fails on GitHub. I'll add an onFailure() clause just in case